.github/workflows/ci.yml

```diff
@@ -11,6 +11,8 @@ jobs:
   build:
     runs-on: ubuntu-20.04
     steps:
+      - name: Startup MySQL service
+        run: sudo /etc/init.d/mysql start
```

Comment:
is this really all we need to do to run mysql for tests?

Reply:
yeah..